mkdocs.yml

```diff
@@ -0,0 +1,70 @@
+site_name: 'ARC'
+site_description: 'Main documentation for ARC'
+repo_url: https://github.com/sourcefuse/arc-docs
+edit_uri: edit/main/docs
```

Comment:
Is it possible to have it disabled for some pages that are non-local (coming from sourceloop repo for example).
As even if someone contributes an edit it'll be overridden in the immediate next sync.